publisher-sdk/src/main/java/com/criteo/publisher/model/SdkFeature.kt

```diff
@@ -0,0 +1,27 @@
+/*
+ *    Copyright 2020 Criteo
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+package com.criteo.publisher.model
+
+/**
+ * Contains codes for features of SDK described in OpenRTB Api Spec
+ */
+@Suppress("MagicNumber")
+enum class SdkFeature(val code: Int) {
```

Comment:
Do you mind renaming to ApiFramework?
That's the name of the enum in ortb specficiation

Reply:
sure